### ISSUE TITLE
Not working on 1.17

### ISSUE THREAD

**Issue by cmod31** [Jannyboy11/InvSee-plus-plus#3]
I don't know if someone works on 1.17 support but it isn't there yet

**Comment by Jannyboy11** [comment]
Correct. I'm working on that. I already have a compiling version, but I need to adjust it still to the changed minecraft internals as spectator inventories are not opening yet.
I'm asking for a bit more patience, hopefully the udpate is done by the end of the week.❤

**Comment by Jannyboy11** [comment]
https://www.spigotmc.org/resources/invsee.82342/update?update=407816
